plugins/stubcontent/class.stubcontent.plugin.php

```diff
@@ -201,7 +201,7 @@ public function addStubContent($type) {
     public function insertContent($content) {
 
         // Don't affect installed forums
-        if (c('Garden.Installed', false) === true) {
+        if (c('Garden.Installed', false) == true) {
```

Comment:
For the sake of compatibility, this check should probably be the same as how Vanilla, itself, actually checks to see if it's installed: with a regular ole truthy check.

https://github.com/vanilla/vanilla/blob/a09bf1558ddbba3635a05f306c31faccfbdf9551/applications/dashboard/controllers/class.setupcontroller.php#L49

What we're doing here is basically a long-form version. We can shorten it a bit.

```suggestion
        if (c('Garden.Installed', false)) {
```

Reply:
As an alternative, you could keep this strict, by matching on the default (and an explicit `false`).

```suggestion
        if (c('Garden.Installed', false) !== false) {
```